### PR TITLE
Advanced styling of headings

### PR DESCRIPTION
**Pull request by planarvoid**
### Fix
In this PR I'm introducing advanced options for styling some elements. For example headings now can be sized and changed each one differently. Headings now support size and color. 

### Test
1. Modify the AztecText style in MainActivity to use `headingNFontSize` or `headingNFontColor` where `N` stands for heading types between `one` and `six`
2. Run the app
3. Notice the heading styling

### Review
@[USER_NAME]

Make sure strings will be translated:

- [x] If there are new strings that have to be translated, I have added them to the client's `strings.xml` as a part of the integration PR.

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -39,7 +39,11 @@
                 android:scrollbars="vertical"
                 android:imeOptions="flagNoExtractUi"
                 aztec:historyEnable="true"
-                aztec:historySize="10"/>
+                aztec:historySize="10"
+                aztec:headingOneFontSize="24sp"
```
Comment (khaykov) [inline app/src/main/res/layout/activity_main.xml]: 24sp is visually very similar (if not the same) to H2. Maybe we can make it a bit bigger to differentiate better in demo app?

**Review by khaykov — COMMENTED** [review]
I haven't touched the Aztec for a while, but from what I can tell the size and color of Heading seems to work as expected. But it looks like something funky is happening with paddings? If you add a text after H1, and convert it to, say, H3, the paddings below H3 shift in weird way. This issue is present to some degree in trunk, with bigger headings, but behavior on the gif bellow is recent development:

[Image from Gyazo](https://gyazo.com/e27607eab74065490120fa533ebee6af)

Comparison using same text sizes:
[Image from Gyazo](https://gyazo.com/f4e0e89a974050cad90d740a0032290a)

Also, it looks like there are some changes around paragraph margin/padding, but I'm not sure what they affect here?

**Comment by planarvoid** [comment]
you're right @khaykov . However, I've tested `trunk` and it's happening there as well so I don't think it's related to these changes. 

I have no idea why the edit text isn't redrawn when the heading is added/removed. I've tried a few approaches and nothing redraws it so I've added a hacky fix which adds and removes a new line when a heading is applied/removed. It works well so let me know what you think!

File: app/src/main/res/layout/activity_main.xml (hunk 1 above)
Comment (planarvoid) [reply app/src/main/res/layout/activity_main.xml]: This was left here by accident, I've removed it 👍

**Comment by khaykov** [comment]
@planarvoid Got it! While fix definitely helps avoid some crazy situations, I think it is a bit too hacky, and I can't really be sure it will not lead to some weird behavior down the line :) I think it would be a good idea to have a dedicated effort on this one, since it looks like other elements apart from heading are also affected by it, as described in #931.

**Comment by planarvoid** [comment]
yeah, it just ignores the updated line height for all the elements. It's just much more prominent for the headings because the line height there is much bigger. I'll revert the change and we can leave it for a separate fix 👍

**Review by khaykov — APPROVED** [review]
Thanks for the changes, @planarvoid ! Looks good to me 👍

**Pull request merged**